external/batch-import.php

```diff
@@ -0,0 +1,63 @@
+<?php
+ini_set('display_errors', 1);
+if (!defined('XHGUI_ROOT_DIR')) {
+    require dirname(__DIR__) . '/src/bootstrap.php';
+}
+
+$sites      = new Xhgui_Sites();
```

Comment:
With a snippet of code which I replaced with lines 3-5 (basically the same thing, but I took this from existing source to be more inline, haven't changed the alignment). Will fix it.